qb/qb.libs.sh

```diff
@@ -76,7 +76,7 @@ check_pkgconf() # $1 = HAVE_$1  $2 = package  $3 = version  $4 = critical error
 	[ "$3" ] && ECHOBUF="$ECHOBUF >= $3"
 
 	[ "$PKG_CONF_PATH" = "none" ] && {
-		eval "HAVE_$1=no"
+		eval "HAVE_$1=pkg"
```

Comment:
This part may also cause issues. Please see: https://github.com/libretro/RetroArch/blob/master/qb/qb.comp.sh#L86

On the test run where I didn't set the PKG_CONF_PATH variable, I dumped the environment variables just before the check_val line for the EGL libraries in qb/config.libs.sh, and it showed these variables set:
```
PKGCONF='/usr/bin/pkg-config'
PKG_CONF_PATH='/usr/bin/pkg-config'
PKG_CONF_USED=' VIDEOCORE'
```
This means that HAVE_EGL etc. will not be set to "pkg", and your check_val function will not do the necessary checking.